backend/src/utils/body.ts

```diff
@@ -29,7 +29,8 @@ export async function readBody<T>(request: Request): Promise<T> {
 			if (out[key2] === undefined) {
 				out[key2] = []
 			}
-			out[key2].push(value)
+			// eslint-disable-next-line @typescript-eslint/no-extra-semi
```

Comment:
for some reason prettier insists in adding this semicolon 😕

Reply:
The bots are fighting each other!

Reply:
yeah! thanks! 🤩